src/components/blocks/SensorSetPointPair/Create.vue

```diff
@@ -23,6 +23,21 @@
         />
       </q-field>
     </q-step>
+    <q-step
```

Comment:
Do users have to put in their own block id? Right now it was auto generated so it could be changed afterwards, which I think is better for the users.

Reply:
Can implement that, but right now it was not generated at all - backend API raised an error over missing 'id' key.

Reply:
Oh! I think my mock API was doing it for me then. Maybe we can prefill the field with a random id so users can immediately update the id if they want. Something like `sensor-setpoint-pair-VEGB123NT4`

Reply:
Sure.